del-geo-core/src/mat3_row_major.rs

```diff
@@ -189,4 +211,53 @@ pub fn svd(m: &[f64; 9], nitr: usize) {
     u[6] = u0[2];
     u[7] = u1[2];
     u[8] = u2[2];
+    let s = [g[0], 0.0, 0.0, 0.0, g[1], 0.0, 0.0, 0.0, g[2]];
+    (u, s, v)
+}
+
+/**
```

Comment:
rust doc is started with `///`

<https://doc.rust-lang.org/rust-by-example/meta/doc.html>

your style is jsdoc <https://en.wikipedia.org/wiki/JSDoc>